src/server/models/tests/mediaquery.test.js

```diff
@@ -0,0 +1,139 @@
+require('dotenv').config();
+require('babel-polyfill');
+const client = require('../db');
+const { getCastsByTerm } = require('../mediaquery');
+
+const dbURL = `${process.env.DB_URL}${process.env.DB_DATABASE}?replicaSet=${process.env.DB_REPLSET}`;
+const testCollectionName = 'mediaquerytests';
+
+beforeAll(async done => {
+  jest.setTimeout(30000);
```

Comment:
@chern This setTimeout() is no longer needed right? Because you added the setup mongodb action in the code coverage file?